### PR TITLE
[FEATURE] Feature Flags, Accounting Service, Billing Service, Tests

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +51,25 @@ sudo /etc/init.d/dnsmasq restart
...
+````
+var features = require('lib/features')
+
+if (features.isEnabled('MY_COOL_FEATURE')) {
+  // New code belongs here
+}
+````
+
+
+Features are enabled at startup using the command line flag -f or --feature. Multiple features can be specified using commas without spaces.
```
Comment (wilsonianb) [inline README.md]: Here we say `--feature` but the example below has `--features`. Which is it?

File: README.md [hunk 2]
```diff
@@ -44,3 +51,25 @@ sudo /etc/init.d/dnsmasq restart
...
+
+````
+codius-host start --features my_cool_feature
+
+codius-host start -f feature_one,feature_two
```
Comment (wilsonianb) [inline README.md]: We could have the example feature names in all caps since our feature names seem to use all caps.

Comment (wilsonianb) [inline README.md]: Nevermind, I see that we call `toUpperCase()` on each feature name

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:

* wilsonianb

As such, I'll attempt to merge and close this pull request.

**Pull request merged**